src/Connection.cpp

```diff
@@ -30,7 +31,8 @@ ssize_t Connection::read(string* buf) {
   if (messagesRead == -1) {
     if (isSkippableError()) {
       // Close the socket and invalidate, then return 0 messages
-      LOG(INFO) << "Closing socket because " << errno << " " << strerror(errno);
+      LOG(ERROR) << "Closing socket because " << errno << " "
```

Comment:
Let's change these back to LOG(INFO).  In the future we will ditch glog for spdlog or another one that does not have issues writing LOG(INFO) in forks